omas/omas_physics.py

```diff
@@ -13,6 +13,31 @@ def add_to__ODS__(f):
 class constants(object):
     e = 1.6021766208e-19
 
+@add_to__ODS__
+def core_profiles_consistent(ods, update=True, use_electrons_density=False):
+    '''
+    Calls all core_profiles consistency functions including
+      - core_profiles_pressures
+      - core_profiles_densities
+      - core_profiles_zeff
+
+    :param ods: input ods
+
+    :param update: operate in place
+
+    :param use_electrons_density:
+            denominator is core_profiles.profiles_1d.:.electrons.density
+            instead of sum Z*n_i in Z_eff calculation
+
+    :return: updated ods
+    '''
+
+    ods_2 = core_profiles_pressures(ods, update=update)
+    ods_2 = core_profiles_densities(ods_2, update=update)
```

Comment:
@bclyons12 `update=update` should only be on the first function call. Subsequent functions should all have `update=False`.

Reply:
@orso82 I think you mean that subsequent calls use `update=True`, right?  The call to `core_profiles_pressures` should have `update=update` to make a new ODS if necessary, while the rest operate in place on that.